.changeset/fluffy-steaks-flow.md

```diff
@@ -0,0 +1,6 @@
+---
+"@logto/schemas": minor
+---
+
+- add Management API hook middleware
+- add DataHook event types
```

Comment:
the changeset looks not right:

- only `@logto/schemas` is involved
- no need for mentioning the middleware change since user will not feel it until we support adding data hooks via Management API?

Reply:
updated